src/components/customer-data/CustomerData.css

```diff
@@ -7,6 +7,7 @@
 
 .customer-data label {
   margin-bottom: 5px;
+  display: flex;
```

Comment:
Let's apply this style only to the new component, cause it's really required only there.
We can add class to the `label` tag inside LabelledDateTimeInput and add this style to it.

Reply:
done